dev/AppNotifications/AppNotification.cpp

```diff
@@ -119,4 +119,17 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         auto lock{ m_lock.lock_exclusive() };
         m_notificationId = id;
     }
+
+    winrt::Microsoft::Windows::AppNotifications::AppNotificationConferencingConfig AppNotification::ConferencingConfig()
+    {
+        auto lock{ m_lock.lock_shared() };
+        return m_conferencingConfig;
+    }
+
+    void AppNotification::ConferencingConfig(winrt::Microsoft::Windows::AppNotifications::AppNotificationConferencingConfig const& conferencingConfig)
+    {
+        THROW_HR_IF(E_NOTIMPL, !AppNotificationConferencingConfig::IsCallingPreviewSupported());
```

Comment:
These APIs should "do nothing" if calling preview is unsupported.  Allow the caller to set/read the value, but don't use it in any way.  And don't throw. :)

Reply:
But how do we inform the user that this feature is not yet supported when they call these APIs?